qutebrowser/browser/commands.py

```diff
@@ -1231,6 +1250,12 @@ def _run_userscript(self, cmd, *args, verbose=False):
             'QUTE_MODE': 'command',
         }
 
+        try:
+            # not sure if int(winId()) is safe to call on every os.
+            env['QUTE_WID'] = str(int(self._tabbed_browser.winId()))
+        except Exception:
+            pass
```

Comment:
Please don't do this. If you actually see an exception happening, catch that exception - if not, just add it to `env` above, and let the tests take care of ensuring it works.